nncf/common/accuracy_aware_training/runner.py

```diff
@@ -277,42 +401,134 @@ def initialize_training_loop_fns(self, train_epoch_fn: Callable[[CompressionAlgo
         self._configure_optimizers_fn = configure_optimizers_fn
         self._dump_checkpoint_fn = dump_checkpoint_fn
         self._tensorboard_writer = tensorboard_writer
-        self._log_dir = log_dir
 
-    def calculate_minimal_tolerable_accuracy(self, uncompressed_model_accuracy: float):
-        if self.maximal_absolute_accuracy_drop is not None:
-            self.minimal_tolerable_accuracy = uncompressed_model_accuracy - self.maximal_absolute_accuracy_drop
-        else:
-            self.minimal_tolerable_accuracy = uncompressed_model_accuracy * \
-                                              (1 - 0.01 * self.maximal_relative_accuracy_drop)
+        self._log_dir = log_dir if log_dir is not None else osp.join(os.getcwd(), 'runs')
+        self._log_dir = configure_accuracy_aware_paths(self._log_dir)
```

Comment:
You should call this only in main process, because it could lead to problems in distributed mode when several process create log_fir independetly. Take a look at https://github.com/openvinotoolkit/nncf/pull/1371